web/src/components/Button/MediumRectangleButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import styled, { DefaultTheme } from 'styled-components';
+
+const Button = styled.button`
+  width: 15vw;
+  border: none;
+  padding: 2vh 2vw 2vh 2vw;
+  border-radius: ${(props) => props.theme.borderRadius};
+  font-size: 2vh;
+  font-weight: 700;
+  color: ${(props) => props.theme.colors.default};
+  background-color: ${(props) => props.theme.colors.primary};
+  transition: 250ms background ease;
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${(props) => props.theme.colors.prhover};
+  }
+`;
+
+interface MediumRectangleButtonProps {
+  displayText: string;
+  type: 'button' | 'submit' | 'reset' | undefined;
+  handleClickButton: () => void;
+  theme: DefaultTheme;
+}
+
+export default function MediumRectangleButton({
```

Comment:
Well `MediumRectangleButton` isn't intuitive. (ex. How large it is?, How much the "medium" is like?) Rather putting options like `size`, `shape` would be better.

Reply:
I just changed to export styled component(without size)